crates/matrix-sdk-base/src/store/mod.rs

```diff
@@ -764,9 +765,9 @@ impl StateChanges {
 
     /// Update the `StateChanges` struct with the given room with a new
     /// `StrippedMemberEvent`.
-    pub fn add_stripped_member(&mut self, room_id: &RoomId, event: StrippedRoomMemberEvent) {
-        let user_id = event.state_key.clone();
-
+    pub fn add_stripped_member(&mut self, room_id: &RoomId, event: Raw<StrippedRoomMemberEvent>) {
+        let user_id =
+            event.get_field("state_key").expect("TODO deser fail").expect("TODO no state_key");
```

Comment:
You're planing on resolving this TODO? 

This method seems to be only used in tests, though it sadly has to be public for the state store integration tests. So having an expect here isn't the end of the world, though we should somehow make it obvious that this isn't really for public consumption.

Reply:
Oh, yes that wasn't intended to still be there for the final PR. Will make the fn return a `Result`.